sudoku_board.h

```diff
@@ -0,0 +1,40 @@
+#pragma
```

Comment:
```suggestion
#pragma once
```

Reply:
In every .h file, writing `#pragma once` makes the header file be included at most once.